helm/ako/crds/nsxalbinfra.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: nsxalbinfrasettings.ako.vmware.com
+spec:
+  conversion:
+    strategy: None
+  group: ako.vmware.com
+  names:
+    kind: NsxAlbInfraSetting
+    listKind: NsxAlbInfraSettingList
+    plural: nsxalbinfrasettings
+    shortNames:
+    - nsxalbinfrasettings
+    - nas
+    singular: nsxalbinfrasetting
+  scope: Cluster
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: NSX ALB Settings is used to select specific Avi controller infra attributes.
+        properties:
+          spec:
+            properties:
+              network:
+                properties:
+                  name:
+                    type: string
+                properties:
+                  rhi:
+                    type: string
+                type: object
+                required:
+                - name
+              seGroup:
```

Comment:
Here it's mentioned as `seGroup` but in the struct json it's `segroup`, so it won't get marshalled properly into the struct. Let's stick with `seGroup`

Reply:
Thx. Done.